### PR TITLE
Website style and structure clean up

### PR DESCRIPTION
**Pull request by RobbieZhuang**
This PR is a cleanup update to match the new design which involves renaming/adding/removing pages on the website as well as making initial changes to the events page.

- **Navbar:** Updated page names to match the new design
   i.e. home, events, news, council, about
- **Event Page:** Added an example events list to the event page 
- Ran some automatic node package fixes
- **Website:** 
   - Update font to Roboto
   - Convert css files to scss
- **About:** Moved the email contact to this page

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -43,5 +44,6 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
-  </body>
-</html>
+</body>
+
+</html>
```
Comment (SDobrik) [inline public/index.html:49]: Why you changing up the spacing?

File: src/App.jsx [hunk 2]
```diff
@@ -19,29 +19,28 @@ class App extends Component {
...
           <Switch>
             <Route exact path='/' component={Home}/>
-            <Route exact path='/platforms' component={Platforms}/>
             <Route exact path='/events' component={Events} />
-            <Route exact path='/about' component={About} />
-            <Route exact path='/contact' component={Contact} />
+            <Route exact path='/news' component={News} />
             <Route exact path='/council' component={Council} />
+            <Route exact path='/about' component={About} />
+            <Route exact path='/platforms' component={Platforms}/>
```
Comment (SDobrik) [inline src/App.jsx:43]: Why the change in ordering here?

File: src/App.scss [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    margin-right: 0;
+    text-align: right;
+}
+
+.nav-link.active {
+    color: #000;
+}
+
+
+/* Site-wide colors */
```
Comment (SDobrik) [inline src/App.scss:56]: uh, aren't the colors at the very top?

File: src/components/EventList/EventList.jsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    body: "Lockdown blues? Join us online to prove your skills, make new friends and play through the night."
+  },
+  {
+    date: "January 1, 2020",
+    title: "Club Penguin Squad Up",
+    body: "Waddle Waddle. Join us for a penguin party. Full SEnd south pole style."
+  }
+]
+
+function Event(props) {
```
Comment (SDobrik) [inline src/components/EventList/EventList.jsx:22]: Why not an ES6 () => {} instead?

File: src/App.jsx (hunk 2 above)
Comment (RobbieZhuang) [reply src/App.jsx:43]: Platforms is hidden in the navbar and I ordered the rest of the routes in navbar order.

File: src/App.scss (hunk 3 above)
Comment (RobbieZhuang) [reply src/App.scss:56]: updated comment to `Site-wide styles`

File: public/index.html (hunk 1 above)
Comment (RobbieZhuang) [reply public/index.html:49]: my linter uwu

File: src/components/EventList/EventList.jsx (hunk 4 above)
Comment (RobbieZhuang) [reply src/components/EventList/EventList.jsx]: is there more to the ES6 pattern, other than using these: `var Event(props) => { return ... }`

Comment (SDobrik) [reply src/components/EventList/EventList.jsx]: meh, but I don't care about it enough to enforce any standard.
